wweval/DESCRIPTION

```diff
@@ -72,7 +72,11 @@ Imports:
     tibble,
     cfaforecastrenewalww,
     cmdstanr (>= 0.7.1),
-    forcats
+    forcats,
```

Comment:
Still needed here:
https://github.com/CDCgov/wastewater-informed-covid-forecasting/blob/23cdf0644b0dec7237b18c0265be49a22b349d12/wweval/R/ms_fig5.R#L380

I think I tried the factor order thing and it didn't work here because its internal to the dataframe rather than an external order we tell it.

Reply:
Thanks. This is the exact use case for `forecats::fct_reorder` so I'm happy to add the dependency